api/libproject/devstart.rb

```diff
@@ -71,7 +71,7 @@ def make_gae_vars(min, max)
     :config_json => "config_perf.json",
     :cdr_versions_json => "cdr_versions_perf.json",
     :featured_workspaces_json => "featured_workspaces_perf.json",
-    :gae_vars => make_gae_vars(20, 20)
+    :gae_vars => make_gae_vars(20, 20, 'all-of-us-rw-perf')
```

Comment:
omit 3rd param, unless you're trying to also test this in perf, in which case should specify an instance class rather thanthe project name

Reply:
done.